### PR TITLE
Fix: "You removed Someone" System message is black in dark mode

### PR DESCRIPTION
**Pull request by billypchan**
### Issues

"You removed Someone" System message's text color is black in dark mode

### Causes

ConversationSystemMessageCell always use black for both light and dark mode.

### Solutions

~~Add a new color scheme .systemMessageForeground which is black is light mode and black in dark mode.~~
Use .textForeground for system message

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift [hunk 1]
```diff
@@ -266,7 +266,9 @@ class ConversationParticipantsChangedSystemMessageCellDescription: ConversationM
     let accessibilityLabel: String? = nil
     
     init(message: ZMConversationMessage, data: ZMSystemMessageData) {
-        let model = ParticipantsCellViewModel(font: .mediumFont, boldFont: .mediumSemiboldFont, largeFont: .largeSemiboldFont, textColor: .black, iconColor: .black, message: message)
+        let color = UIColor.from(scheme: .systemMessageForeground)
```
[the remark this answers is not in the document]
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift]: .textForeground's light mode color is not .black. and there is no existing color scheme with text color pure black.

[the remark this answers is not in the document]
Comment (mikeger) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift]: @billypchan I don't think it should be pure black, textForeground should be ok?

[the remark this answers is not in the document]
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift]: I checked the design spec and it should be same as other message's color.

**Comment by billypchan** [comment]
@typfel updated snapshots affected by this PR, e.g. 
https://github.com/wireapp/wire-ios/pull/2981/files?short_path=317d86c#diff-317d86cbe2724c81a0fb1bd16577e085

**Pull request merged**